src/main/java/com/CezaryZal/manager/creator/HttpEntityByBodyAndToken.java

```diff
@@ -0,0 +1,22 @@
+package com.CezaryZal.manager.creator;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.util.MultiValueMap;
+
+@Service
+public class HttpEntityByBodyAndToken {
+
+    private HeadersByToken headersByToken;
+
+    @Autowired
+    public HttpEntityByBodyAndToken(HeadersByToken headersByToken) {
+        this.headersByToken = headersByToken;
+    }
+
+    public HttpEntity<Object> createHttpEntity (Object body, String clearToken){
+        MultiValueMap<String, String> headers = headersByToken.createHeadersByToken(clearToken);
+        return new HttpEntity<>(body, headers);
+    }
+}
```

Comment:
Nazwa klasy totalnie nic mi nie mówi za co jest odpowiedzialna. 
To chyba też powinno wylecieć, bo to do tego requestu

Reply:
Obecnie jak powyżej, zrobiłem z niej metodę statyczną